third_party/move/tools/move-package/src/resolution/resolution_graph.rs

```diff
@@ -107,14 +108,33 @@ impl ResolvingGraph {
         }
         let mut resolution_graph = Self {
             root_package_path: root_package_path.clone(),
-            build_options,
+            build_options: build_options.clone(),
             root_package: root_package.clone(),
             graph: DiGraphMap::new(),
             package_table: BTreeMap::new(),
         };
 
+        let override_std = if let Some(version_str) = &build_options.override_std {
+            if let Some(version) = StdVersion::from_rev(version_str) {
+                Some(version)
+            } else {
+                bail!(
+                    "Invalid verion for standard libraries to override: {}",
+                    version_str
```

Comment:
Could we say here the expected strings? See also my remark about `ValueEnum` derive below.